client/src/contexts/MinutesContext.jsx

```diff
@@ -0,0 +1,44 @@
+import { createContext, useState } from "react";
+
+const MinutesContext = createContext();
+
+export const MinutesContextProvider = (props) => {
+  const [minutes, setMinutes] = useState({
+    name: "",
+    colors: {
+      primary: "#000000",
+      secondary: "#FFFFFF",
+    },
+    segments: [
+      {
+        name: "Agenda",
+        content: "",
+      },
+      {
+        name: "Decisions",
+        content: "",
+      },
+      {
+        name: "",
+        content: "",
+      },
+    ],
+    startTime: null,
+    signatures: [],
+  });
+
+  const updateMinutes = (newMinutesData) => {
+    setMinutes((prevMinutes) => ({
+      ...prevMinutes,
+      ...newMinutesData,
+    }));
+  };
+
+  return (
+    <MinutesContext.Provider value={{ minutes, updateMinutes }}>
```

Comment:
Set this as a tuple (array with two values for example like this: [minutes, setMinutes])  to match the conventions of react